qube/create/templates/lib/lib_java/{{ cookiecutter.project_slug }}/src/main/groovy/life/qbic/example/Example.groovy

```diff
@@ -0,0 +1,5 @@
+package life.qbic.example
```

Comment:
That's a groovy file inside a Java template.

lib_java has no groovy support (yet!).

Reply:
sure it has, via the parent pom ;)

Reply:
Ahh, it's part of the parent pom.

In my (old) QBiC projects I included it manually :)

Still, for consistency I would prefer a .java here, but the choice is yours :)

Reply:
my IDE does not recognize that groovy is allowed/ included in the projects.